src/declarativedbusinterface.cpp

```diff
@@ -415,7 +416,8 @@ void DeclarativeDBusInterface::componentComplete()
 
 void DeclarativeDBusInterface::pendingCallFinished(QDBusPendingCallWatcher *watcher)
 {
-    QJSValue callback = m_pendingCalls.take(watcher);
+    QPair<QJSValue, QJSValue> callbacks = m_pendingCalls.take(watcher);
+    QJSValue callback = callbacks.first;
```

Comment:
It does appear like a check that should never trigger, only callable callbacks are inserted into m_pendingCalls. Can anyway move the check after calling the error handler.